app/src/main/java/com/example/musclenerds/ui/Adapters/ExerciseAdapter.java

```diff
@@ -0,0 +1,87 @@
+package com.example.musclenerds.ui.Adapters;
+
+import android.app.Dialog;
+import android.app.Fragment;
+import android.content.Context;
+import android.graphics.Color;
+import android.os.Bundle;
+import android.os.Parcelable;
+import android.view.LayoutInflater;
+import android.view.MotionEvent;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageButton;
+import android.widget.TextView;
+
+import androidx.recyclerview.widget.RecyclerView;
+
+import com.example.musclenerds.MainActivity;
+import com.example.musclenerds.R;
+import com.example.musclenerds.ui.models.Exercise;
```

Comment:
As per @robDevs mention, we need to hook this exercise view functionality up to the exercises we will be storing in the db. This is a hardcoded hack for now, and that is fine - needs to be fixed later on.